packages/shared/src/util/features.ts

```diff
@@ -114,3 +116,54 @@ export function validateFeatureValue(
 
   return value;
 }
+
+type StaleFeatureReason =
+  | "error"
+  | "draft-state"
+  | "no-active-envs"
+  | "no-rules"
+  | "no-active-exps"
+  | "all-exps-onesided";
+
+export function isFeatureStale(
+  feature: FeatureInterface,
+  linkedExperiments: ExperimentInterface[] | undefined = []
+): { stale: boolean; reason?: StaleFeatureReason } {
+  if (feature.linkedExperiments?.length && !linkedExperiments.length) {
+    // eslint-disable-next-line no-console
+    console.error("isFeatureStale: linkedExperiments not provided");
+    return { stale: false, reason: "error" };
+  }
+
+  const twoWeeksAgo = subWeeks(new Date(), 2);
+  // TODO why is dateUpdated a string?
+  const stale = new Date(feature.dateUpdated) < twoWeeksAgo;
+
+  if (!stale) return { stale };
+
+  if (feature.draft) return { stale, reason: "draft-state" };
```

Comment:
This will return `stale: true` when it's a draft.  I think we want the inverse:

```ts
  if (feature.draft) return { stale: false, reason: "draft-state" };
```

Also, might want to have some date checks to make sure the draft is recent.  This will all change in my drafts refactor PR, so you can ignore that part for now until that PR is merged.

Reply:
I updated this to mark stale if `draft.dateUpdated` is older than two weeks, but will revisit after your feature draft PR is merged